trax/data/tf_inputs.py

```diff
@@ -1107,6 +1112,42 @@ def bert_double_sentence_preprocess(batch):
     yield value_vector, segment_embs, segment_embs, label, np.int64(1)
 
 
+def mask_random_tokens(batch, vocab_size=30522, masking_prob=0.15, merge_with_loss=True):
+  """
+  Follows standard procedure of masking in BERT:
+  `masking_prob` random tokens are selected out of which:
+    80% are replaced with [MASK] token
+    10% are replaced with random token
+    10% are leaved with original token unchanged
+
+  ids of the tokens are from original lowercase vocab
+  """
+
+  for token_ids, *row_rest in batch:
```

Comment:
Could you comment either in the docstring or comments what `batch` is, like type, shape etc